src/widgetastic_patternfly5/components/duallistselector.py

```diff
@@ -0,0 +1,143 @@
+
+    @property
+    def _right_elements(self):
+        return self.browser.elements(self.ITEMS, parent=self._right_list)
+
+    @property
+    def _selected_left_elements(self):
+        return self.browser.elements(self.SELECTED_ITEMS, parent=self._left_list)
+
+    @property
+    def _selected_right_elements(self):
+        return self.browser.elements(self.SELECTED_ITEMS, parent=self._right_list)
+
+    def move_all_items_right(self):
+        self.move_all_right.click()
+
+    def move_all_items_left(self):
+        self.move_all_left.click()
+
+    def move_selected_items_right(self):
+        self.move_selected_right.click()
+
+    def move_selected_items_left(self):
+        self.move_selected_left.click()
+
+    def read(self, selected_only=False):
+        """Read items on left and right sides, if selected_only = True, read only selected items"""
+        data = dict()
+
+        if selected_only:
+            right_elements = self._selected_right_elements
+            left_elements = self._selected_left_elements
+        else:
+            right_elements = self._right_elements
+            left_elements = self._left_elements
+
+        data[self._left_title] = [el.text for el in left_elements]
+        data[self._right_title] = [el.text for el in right_elements]
+        return data
+
+    def reset_selected(self, left_items=True):
+        if left_items:
+            _elements = self._left_elements
+        else:
+            _elements = self._right_elements
+
+        for element in _elements:
+            if element.get_attribute("aria-selected") == "true":
+                element.click()
+
+    def select(self, items, left_items=True):
+        """Select certain items based on text. Defaults to select left_items"""
+        if left_items:
+            _elements = self._left_elements
+        else:
+            _elements = self._right_elements
+
+        text_item_map = {self.browser.text(el): el for el in _elements}
+        for item in items:
+            el = text_item_map.get(item)
+            if not el:
+                raise ValueError("item not available. select proper one like raise an error")
+            self.browser.click(el)
+
+    def select_and_move(self, items, left_items=True):
+        """Select and move certain items based on text. Defaults to select
+        left_items and move to right"""
+        self.select(items, left_items=left_items)
+
+        if left_items:
+            self.move_selected_items_right()
+        else:
+            self.move_selected_items_left()
+
+
+class DualListSelector(BaseDualListSelector, GenericLocatorWidget):
+    pass
+
+
+class SearchDualListSelector(DualListSelector):
+    INPUT = "//input"
+
+    def search(self, value, left_column=True):
+        """Fills a Dual list selector with the supplied value depending on column."""
+        if left_column:
+            text_input_widget = TextInput(self, locator=self.AVAILABLE + self.INPUT)
```

Comment:
really they added locator strings :) 
Lets ignore this but we need to fix it later :)